docs/framework/whats-new/whats-new-in-accessibility.md

```diff
@@ -240,9 +240,9 @@ Starting with the .NET Framework 4.7.1, developers of accessibility technology t
  
 - The <xref:System.Windows.Forms.DataGridViewCheckBoxCell> now supports the [toggle pattern](../ui-automation/implementing-the-ui-automation-toggle-control-pattern.md).
  
-- The <xref:System.Windows.Forms.ToolStripItem> control supports the <xref:System.Windows.Automation.AutomationElement.Name> property and the [expand/collapse pattern](../ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern.md).
+- The <xref:System.Windows.Forms.ToolStripItem> control supports the <xref:System.Windows.Automation.AutomationElement.AutomationElementInformation.Name> property and the [expand/collapse pattern](../ui-automation/implementing-the-ui-automation-expandcollapse-control-pattern.md).
 
-- The <xref:System.Windows.Forms.NumericUpDown> and <xref:System.Windows.Forms.DomainUpDown> controls support the <xref:System.Windows.Automation.automationElement.Name> property.
+- The <xref:System.Windows.Forms.NumericUpDown> and <xref:System.Windows.Forms.DomainUpDown> controls support the <xref:System.Windows.Automation.AtomationElement.AutomationElementInformation.Name> property.
```

Comment:
AtomationElement is probably gonna fail. Shouldn't be Auto...?

Reply:
Yes, it should have been "Auto...". I can't believe I missed that, but I've corrected it. Thanks, @mairaw.